apps/site/assets/ts/app/mobile-menu.ts

```diff
@@ -81,6 +81,15 @@ const setupMobileMenu = (): void => {
 
   // Changing viewport size closes
   window.addEventListener("resize", closeMenu);
+
+  // removes focus outline in Safari from open accordions
+  document
+    .querySelectorAll(".m-menu__content .js-focus-on-expand")
+    .forEach(acc => {
+      acc.addEventListener("focus", function undoOutline(this: HTMLElement) {
+        this.style.outline = "none";
```

Comment:
I think you can add this to the focus call [here](https://github.com/mbta/dotcom/blob/master/apps/site/assets/js/collapse.js#L20) and have the same effect. I'm not sure if it's wanted to have the focus outline for any other case though

Reply:
Yeah, the particular component this is operating on (the accordion) is reused in many places across the website! But in this particular case the accordion content has focusable content inside, which has its own visible focus outline, so the default focus outline is redundant.